### ISSUE TITLE
registerRoutes not working with forward slash on windows

### ISSUE THREAD

**Issue by FrankVisuals** [lorisleiva/laravel-actions#92]
When calling 
```
Actions::registerRoutes('app/Actions/Test');
```
i have the issue

```
ErrorException: class_parents(): Class App\C:\<my-path>\app\Actions\Test\Login does not exist and could not be loaded 
```

The issue here is that the path was not parsed correctly for me (using Windows). It only works when calling 

```
Actions::registerRoutes('app\\Actions\\Test');
```

**Comment by lorisleiva** [comment]
Hi there 👋 

Thanks for raising this and having a go at it.

If I remember correctly, I got heavily inspired with how Laravel auto-registers the Commands directory for this feature.

Does this work well for your on Windows? If so I might have missed something there. I'd really appreciate your help in this since I have no Windows computer to test that. 😕 

Thanks again. 🙂

**Comment by FrankVisuals** [comment]
Hey! Thanks for the fast response. I tried to provide a PR with a fix, however it seems to fail on Linux (tests) - it worked for me (https://github.com/lorisleiva/laravel-actions/pull/93). I can try to give you as many details as possible.

I narrowed it down to the `ActionManager` class where `Str::after($pathname, realpath(app_path()) . DIRECTORY_SEPARATOR)` results in `"C:\<path to my project>\app/Actions/Test\CustomLogin.php"` when being called like `Actions::registerRoutes('app/Actions/Test');`

The actual result here should be `Actions\Test\CustomLogin.php` - which works when I use `Actions::registerRoutes('app\\Actions\\Test');`

Hope that helps!

**Comment by lorisleiva** [comment]
Hey, yeah I saw but PR #93 will only work on Windows. The same goes when using backslashes in your path — i.e. `Actions::registerRoutes('app\\Actions\\Test');`. Even in your case you'll probably end up deploying this code on a linux server which will fail at this point.

I've double-checked [the `load` method on the Laravel Console Kernel](https://github.com/laravel/framework/blob/b69db31d47342f41878c20cbdce35edc8ea56147/src/Illuminate/Foundation/Console/Kernel.php#L208-L236) and I'm using the exact same process to discover files in an array of directories.

The only difference I see is [they use an absolute path](https://github.com/laravel/laravel/blob/06d967a4c72be2ec71a0efd89cc2a3c113cf6da5/app/Console/Kernel.php#L35-L40) by default when loading the `Commands` directory inside `laravel/laravel`. However, they still use a forward slash to define the rest of the path.

```php
protected function commands()
{
    $this->load(__DIR__.'/Commands');

    require base_path('routes/console.php');
}
```

Could you try to use the same approach when registering your routes? Something like this:

```php
// Assuming you're running this inside app/Providers/AppServiceProvider.
Actions::registerRoutes(__DIR__.'/../Actions/Test');
```

**Comment by FrankVisuals** [comment]
I reopened the PR and made a change that seems to work now. The issue for me was caused by adding the delimitor which wasn't matching my path. So instead of adding the delimitor to the replace clause I've removed the first character of the resulting string.

I've also provided two tests for this to check to work with both - backslash and frontslash :) 

I hope that helps!

PS: Using `Actions::registerRoutes(__DIR__.'/../Actions/Test');` did not work for me

**Comment by lorisleiva** [comment]
Hi! As mentioned in the PR, I've fixed this by using `getRealPath` instead of `getPathname`.

This has been released as part of [`v2.1.1`](https://github.com/lorisleiva/laravel-actions/releases/tag/v2.1.1). 🚀 

Let me know if that all works fine for you. Thanks again for raising this! 🍺
